src/GodelTech.Microservices.Core/GodelTech.Microservices.Core.csproj

```diff
@@ -10,7 +10,7 @@
     <Authors>Andrei Salanoi</Authors>
     <RepositoryUrl>https://github.com/GodelTech/GodelTech.Microservices.Core</RepositoryUrl>
     <PackageIconUrl>https://www.gravatar.com/avatar/839234621070de51e7b9cabd5ceee8fe?s=64</PackageIconUrl>
-    <VersionPrefix>1.3.0</VersionPrefix>
+    <VersionPrefix>1.3.1</VersionPrefix>
```

Comment:
@Pavel-Lapatin i think we can revert this change because we are not going to re-build package. Our changes are not related to package. These are infrastructure changes.

Reply:
Change was reverted